conans/model/editable_cpp_info.py

```diff
@@ -0,0 +1,82 @@
+# coding=utf-8
+import configparser
+import ntpath
+import os
+import posixpath
+from collections import defaultdict
+
+from conans.client.tools.files import load
+
+
+class EditableCppInfo(object):
+    WILDCARD = "*"
+    cpp_info_dirs = ['includedirs', 'libdirs', 'resdirs', 'bindirs']
+
+    def __init__(self, data, uses_namespace):
+        self._data = data
+        self._uses_namespace = uses_namespace
+
+    @classmethod
+    def create(cls, filepath_or_content, require_namespace):
+        _parse_func = cls.parse_file if os.path.exists(filepath_or_content) else cls.parse_content
+        data = _parse_func(filepath_or_content, require_namespace=require_namespace)
+        return EditableCppInfo(data, uses_namespace=require_namespace)
+
+    @classmethod
+    def parse_file(cls, filepath, require_namespace):
+        return cls.parse_content(content=load(filepath), require_namespace=require_namespace)
+
+    @classmethod
+    def parse_content(cls, content, require_namespace):
+        """ Returns a dictionary containing information about paths for a CppInfo object: includes,
+        libraries, resources, binaries,... """
+
+        parser = configparser.ConfigParser(allow_no_value=True, delimiters=('#', ))
+        parser.optionxform = str
+        parser.read_string(content)
+
+        if not require_namespace:
+            ret = {k: [] for k in cls.cpp_info_dirs}
+            for section in ret.keys():
+                if section in parser:
+                    ret[section] = parser[section]
+            return ret
+        else:
+            ret = defaultdict(lambda: {k: [] for k in cls.cpp_info_dirs})
+            for section in parser:
+                if ':' in section:
+                    namespace, key = section.split(':', 1)
+                    if key in cls.cpp_info_dirs:
+                        ret[namespace][key] = parser[section]
+            return ret
+
+    @staticmethod
+    def _work_on_item(value, base_path, settings, options):
+        value = value.replace('\\', '/')
+        isabs = ntpath.isabs(value) or posixpath.isabs(value)
```

Comment:
I still see this line as weird, need to understand why ``ntpath.isabs(value) or posixpath.isabs(value)``

Reply:
It has been commented before (conversation still opened)

Reply:
Can't find the conversation.